src/custom/CustomTooltip/customTooltip.tsx

```diff
@@ -13,6 +15,10 @@ type CustomTooltipProps = {
   bgColor?: string;
 } & Omit<TooltipProps, 'title' | 'onClick'>;
 
+const StyledFontWrapper = styled(Typography)(() => ({
+  fontFamily: ['Qanelas Soft Regular'].join(',')
+}));
+
```

Comment:
use the typography from the theme. take a theme as a props in the typography

Reply:
yes i already told yesterday pass variant not font family

Reply:
Hello @amitamrutiya @sudhanshutech here we can't use this approach:

const StyledFontWrapper = styled(Typography)(({theme}) => ({
  typography: theme.typography.textH2Regular,
}));

cause the styled component is being imported from the mui library, and the theme tokens are also coming from mui Typography api. So they'll take the default font of MUI which is Roboto.
I've already tried that.